package.json

```diff
@@ -1,15 +1,10 @@
 {
   "scripts": {
-    "layer0:init": "layer0 init",
-    "layer0:dev": "layer0 dev",
```

Comment:
Why pull all the Layer0 commands?

Reply:
Dunno what happened to me, puttin em back.